README.rst

```diff
@@ -551,6 +555,46 @@ the "h" and "hello" messages will be delivered to subscribers:
     }
   }
 
+Message throttling
+------------------
+
+Messages published by services can be throttled by specifying the time in
+seconds using `_throttle` in the published message. Note that the first message
+will be sent immediately, and the last message will be sent eventually.
```

Comment:
Just to clarify, for a constant stream of messages that are coming in shorter than the throttle period.  The client will receive the first message immediately, then a message every throttle period until the stream ends and then the final message will be sent no longer than the throttle value after it was received.